### ISSUE TITLE
Remove ndjson scripts from existing pathogen repos

### ISSUE THREAD

**Issue by victorlin** [nextstrain/ingest#28]
Follow-up to #1. Copying some useful text from the Google doc linked there:

### csv-to-ndjson / tsv-to-ndjson

No longer needed as this can be done with augur curate passthru.

We can remove this script from the following repos (found with [code search](https://github.com/search?type=code&q=org%3Anextstrain+path%3A**%2F*sv-to-ndjson*)) as we update their workflows to use augur curate. 

- [ ] hepatitsB
- [ ] monkeypox
- [ ] ncov-ingest
- [ ] Rsv
- [ ] Dengue (won't show up in code search as it's on a branch)
- [ ] Zika and Ebola (won't show up in code search as it's on a branch)

### fasta-to-ndjson

No longer needed as this can be done with a combination of augur parse and augur curate passthru. 

We can remove this script from the following repos (found with [code search](https://github.com/search?type=code&q=org%3Anextstrain+path%3A**%2Ffasta-to-ndjson*)) as we update their workflows.

- [ ] monkeypox
- [ ] Rsv
- [ ] Dengue (won't show up in code search as it's on a branch)
- [ ] Zika and Ebola (won't show up in code search as it's on a branch)

### ndjson-to-tsv-and-fasta

No longer needed as this can be done with augur curate passthru. 

We can remove this script from the following repos (found with [code search](https://github.com/search?type=code&q=org%3Anextstrain+path%3A**%2Fndjson-to-tsv-and-fasta)) as we update their workflows.

- [ ] monkeypox 
- [ ] Rsv
- [ ] Dengue (won't show up in code search as it's on a branch)
- [ ] Zika and Ebola (won't show up in code search as it's on a branch)

**Comment by joverlee521** [comment]
I consider this slightly blocked by a [bug in `augur curate passthru`](https://github.com/nextstrain/augur/issues/1312).
